### ISSUE TITLE
Add option for Squiz.WhiteSpace.OperatorSpacing to invert for declare

### ISSUE THREAD

**Issue by bradynpoulsen** [squizlabs/PHP_CodeSniffer#1163]
It would be really nice to use the following configuration to indicate that declare statements should match `declare(strict_types=1);`

``` xml
<rule ref="Squiz.WhiteSpace.OperatorSpacing">
    <properties>
        <property name="invertDeclares" value="true"/>
    </properties>
</rule>
```

**Comment by aik099** [comment]
Does `Squiz.WhiteSpace.OperatorSpacing` sniff really care about `declare(strict_types=1);`? I thought it only checks spacing before/after operators (e.g. `=`, `+`, etc.).

**Comment by bradynpoulsen** [comment]
@aik099 You are correct, that includes the operator inside of declares. So `phpcbf` would covert that to `declare(strict_types = 1);`

**Comment by aik099** [comment]
I see now. I think we should simply ignore contents of `declare(...)` function call without an option even.

**Comment by bradynpoulsen** [comment]
True, I guess it could be the responsibility of like a `DeclareOperatorSpacing` sniff to choose a configuration for `declare`'s
